HospitalSystem.ConsoleUI/Secretary/EquipUI.cs

```diff
@@ -212,18 +213,19 @@ public void EquipmentTable(List<EquipmentBatch> equipments, string header)
 
         foreach(var equipment in equipments)
         {
-            if (equipments.Count == 0)
-            {
-            System.Console.WriteLine(String.Format("|[{0,-8}]|[{1,-9}]|[{2, -5}]|", equipment.RoomLocation, equipment.Name, equipment.Count));
-            }
-            else
-            {
-            System.Console.WriteLine(String.Format("| {0,-8} | {1,-9} | {2, -5} |", equipment.RoomLocation, equipment.Name, equipment.Count));
+            List<Room> stocks = _hospital.RoomService.GetStocks().ToList();
+            if(!stocks.Any(stock => stock.Location == equipment.RoomLocation)){
+                if (equipment.Count == 0)
+                {
+                System.Console.WriteLine(String.Format("|[{0,-8}]|[{1,-9}]|[{2, -5}]|", equipment.RoomLocation, equipment.Name, equipment.Count));
+                }
+                else
+                {
+                System.Console.WriteLine(String.Format("| {0,-8} | {1,-9} | {2, -5} |", equipment.RoomLocation, equipment.Name, equipment.Count));
```

Comment:
Why are these two nearly identical lines in different blocks? Do they even need to be different?

Reply:
How are they emphasized?

Reply:
With [] brackets

Reply:
professor: "Specijalno je označena roba koje nema u prostorijama da bi 
je sekretar lakše uočio. "